activemq-mqtt/pom.xml

```diff
@@ -293,6 +294,70 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+     <id>parallel-tests</id>
+     <activation>
+       <property>
+         <name>parallel-tests</name>
+       </property>
+     </activation>
+     <build>
+       <plugins>
+         <plugin>
+           <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
side note: best practise would be to also set the groupId to qualify the plugin, here you do use a legacy old maven style feature which can lead to conflicts randomly depending user configuration.

Reply:
I'd agree and had it in the first place to be honest. 
I always put the groupId for consistency and avoid having sometimes implicit versus explicit values.
Maven 4 or 4.1 now prints a warning when there is a duplicate plugin defined. I'm not sure it would work if you have one using implicit and another one using explicit.